### ISSUE TITLE
Polygons with many points cause app to become unresponsive

### ISSUE THREAD

**Issue by turtle0x1** [heartexlabs/label-studio#4617]
**Describe the bug**
Some models return polygons with 6000 or more points in, the app/browser becomes un-responsive when loading them / clicking between images.

As a workaround I should probably reduce the number of polygon points.

**To Reproduce**
Add a polygon with a large number points from a prediction / pre-annotation.

**Expected behavior**
The app doesn't become unresponsive

**Environment (please complete the following information):**
 - OS: Firefox / Chrome
 - Label Studio Version [1.8.0]

**Comment by AymanTalkhani** [comment]
@turtle0x1 Have you tried using [InstaLabel](https://www.instalabel.ai/)? It automates most of your polygon masks and bounding box labeling with custom AI. All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. There's even a crowdsourcing option to correct AI predictions for verified ground truth labels. Could be useful for a lot of custom computer vision labeling use cases if you're on the lookout.
